lib/color-formatters.spec.js

```diff
@@ -1,64 +1,56 @@
 'use strict';
 
-const assert = require('assert');
+const { test, given, forCases } = require('sazerac');
 const {
   coveragePercentage,
   colorScale,
   age
 } = require('./color-formatters');
 
 describe('Color formatters', function() {
-  it('should step appropriately', function() {
-    const byPercentage = colorScale([Number.EPSILON, 80, 90, 100]);
-    assert.equal(byPercentage(-1), 'red');
-    assert.equal(byPercentage(0), 'red');
-    assert.equal(byPercentage(0.5), 'yellow');
-    assert.equal(byPercentage(1), 'yellow');
-    assert.equal(byPercentage(50), 'yellow');
-    assert.equal(byPercentage(80), 'yellowgreen');
-    assert.equal(byPercentage(85), 'yellowgreen');
-    assert.equal(byPercentage(90), 'green');
-    assert.equal(byPercentage(100), 'brightgreen');
-    assert.equal(byPercentage(101), 'brightgreen');
-  });
+  const byPercentage = colorScale([Number.EPSILON, 80, 90, 100]);
 
-  it('should have parity with coveragePercentage', function() {
-    const byPercentage = colorScale([Number.EPSILON, 80, 90, 100]);
-    assert.equal(byPercentage(-1), coveragePercentage(-1));
-    assert.equal(byPercentage(0), coveragePercentage(0));
-    assert.equal(byPercentage(0.5), coveragePercentage(0.5));
-    assert.equal(byPercentage(1), coveragePercentage(1));
-    assert.equal(byPercentage(50), coveragePercentage(50));
-    assert.equal(byPercentage(80), coveragePercentage(80));
-    assert.equal(byPercentage(85), coveragePercentage(85));
-    assert.equal(byPercentage(90), coveragePercentage(90));
-    assert.equal(byPercentage(100), coveragePercentage(100));
-    assert.equal(byPercentage(101), coveragePercentage(101));
+  test(byPercentage, () => {
+    forCases([
+      given(-1).expect('red'),
+      given(0).expect('red'),
+      given(0.5).expect('yellow'),
+      given(1).expect('yellow'),
+      given(50).expect('yellow'),
+      given(80).expect('yellowgreen'),
+      given(85).expect('yellowgreen'),
+      given(90).expect('green'),
+      given(100).expect('brightgreen'),
+      given(101).expect('brightgreen'),
+    ]);
```

Comment:
Why use `forCases` if you're expecting a different output value for each input? Wouldn't it be better to group per colour?

Reply:
Good idea. I'll update that.

Reply:
They seem harder to read grouped, so I'm removing the `forCases()` call instead.
```js
  test(byPercentage, () => {
    given(-1).expect('red');
    given(0).expect('red');
    given(0.5).expect('yellow');
    given(1).expect('yellow');
    given(50).expect('yellow');
    given(80).expect('yellowgreen');
    given(85).expect('yellowgreen');
    given(90).expect('green');
    given(100).expect('brightgreen');
    given(101).expect('brightgreen');

    forCases([-1, 0, 0.5, 1, 50, 80, 85, 90, 100, 101]
      .map(v => given(v).expect(coveragePercentage(v))))
      .should("return '%s', for parity with coveragePercentage()");
  });
```